debezium-core/src/main/java/io/debezium/relational/RelationalSnapshotChangeEventSource.java

```diff
@@ -581,7 +585,11 @@ private void doCreateDataEventsForTable(ChangeEventSourceContext sourceContext,
 
             LOGGER.info("\t Finished exporting {} records for table '{}' ({} of {} tables); total duration '{}'",
                     rows, table.id(), tableOrder, tableCount, Strings.duration(clock.currentTimeInMillis() - exportStart));
-            snapshotProgressListener.dataCollectionSnapshotCompleted(partition, table.id(), rows);
+            snapshotProgressListener.dataCollectionSnapshotCompleted(snapshotContext.partition, table.id(), rows);
+            LOGGER.info("notifyCompletedTable: " + table.id().table());
+            notificationService.initialSnapshotNotificationService().notifyCompletedTable(snapshotContext.partition,
+                    snapshotContext.offset,
+                    table.id().table());
         }
         catch (SQLException e) {
             throw new ConnectException("Snapshotting of table " + table.id() + " failed", e);
```

Comment:
Can be possible to add a status for `notifyCompletedTable` as for the incremental snapshot? Take a look to `TableScanCompletionStatus`.

Reply:
Ok, I can add the status [SQL_EXCEPTION, SUCCEEDED]. Is this enough or do we also need the other states like EMPTY, SKIPPED, NO_PRIMARY_KEY and UNKNOWN_SCHEMA?

Reply:
I think that its fine to have `SQL_EXCEPTION, SUCCEEDED` for now.